doc/nrf/app_dev/device_guides/nrf54l/kmu_provision.rst

```diff
@@ -0,0 +1,79 @@
+.. _ug_nrf54l_developing_provision_kmu:
+
+
+nRF54L15 KMU provisioning
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF54L15 DK is equipped with Hardware Key Management Unit (KMU), that requires provisioning when in use.
+The |NCS| provides a west command, ``ncs-provision``, allowing to upload keys to the device though the Serial Write Debug (SWD) interface.
+
+Prerequisites
+*************
+
+First, ensure that the nrfprovision script is installed.
+It should install automatically during the setup of the |NCS| working environment.
+If it was not installed, or if you wish to install it manually, run the following command:
+
+.. parsed-literal::
+   :class: highlight
+
+    pip install nrfprovision==0.9.0 --extra-index-url https://files.nordicsemi.com/artifactory/api/pypi/nordic-pypi/simple
+
+
+Key generation
+**************
+
+If you need a new key, you can generate it using imgtool or another tool that produces the required kind and format of key.
+For instructions on how to generate a key, see the :doc:`imgtool page in the MCUboot documentation<mcuboot:imgtool>`.
+See the following example for generating a private key:
+
+.. parsed-literal::
+   :class: highlight
+
+   ./bootloader/mcuboot/scripts/imgtool.py keygen -k my_ed25519_priv_key.pem -t ed25519
```

Comment:
this assumes a patch, if thy are using ncs or have installed it using pip then imgtool should be available globally without needing a path so this should be removed